### PR TITLE
revamp: Add guest house booking component

### PR DESCRIPTION
**Pull request by adityachopra29**
Screenshot 2024-03-26 at 9 41 35 PM

@beingnoble03

### CODE REVIEW

**Comment by adityachopra29** [comment]
@beingnoble03

File: src/components/room-new/index.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+                  </div>
+                </div>
+                )
+            })}
+            <span className={tailwindWrapper('mb-4')}>
+            Want to book a guest house for your parents or relative<br/>
+            Book here
+            </span>
+            <button className={tailwindWrapper('bg-[#133BC5] self-start rounded-md text-white px-28 py-3 mb-10')} onClick={() => {
+              window.alert("book button clicked")
```
Comment (beingnoble03) [inline src/components/room-new/index.js:80]: remove `window.alert("book button clicked")`

File: src/components/room-new/index.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+            <hr/>
+            { bookingRequests.results.map((request, index) => {
+              return(
+                <div className={tailwindWrapper('flex item-center justify-around font-extralight text-xs')}>
+                  <div className={tailwindWrapper('max-w-20')}>{request.bookedBy}</div>
+                  <div className={tailwindWrapper('max-w-20')} >To be decided</div>
+                  <div className={tailwindWrapper('max-w-20')}>{request.requestedFrom}-{request.requestedTill}</div>
+                  <div className={tailwindWrapper('max-w-20')} >
+                  Pending
+                  </div>
```
Comment (beingnoble03) [inline src/components/room-new/index.js:71]: what does `To be decided` mean here?

and, `Pending` is hardcoded here?

**Review by beingnoble03 — CHANGES_REQUESTED** [review]

File: src/components/room-new/index.js (hunk 2 above)
Comment (adityachopra29) [reply src/components/room-new/index.js:71]: This is because there is no backend url from which this data is being delivered...as discussed

**Pull request closed without merging**